Translations/pt_BR.po

```diff
@@ -1379,7 +1370,7 @@ msgstr "A energia vem à custa de sabedoria"
 #. TRANSLATORS: Shrine Text. Keep atmospheric. :)
 #: Source/error.cpp:57
 msgid "Riches abound when least expected"
-msgstr "Riquezas abundam quando menos se espera"
+msgstr "As riquezas abundam quando menos se espera"
```

Comment:
I disagree with this in the sense that the original without the article sounds more formal, which makes more sense for shrine text.

Reply:
so this?
```suggestion
msgstr "Riquezas abundam quando menos se espera"
```